app-editors/neovim/neovim-0.11.0-r1.ebuild

```diff
@@ -113,7 +113,7 @@ src_install() {
 
 	# conditionally install a symlink for nvimpager
 	if use nvimpager; then
-		dosym ../share/nvim/runtime/macros/less.sh /usr/bin/nvimpager
```

Comment:
Needs a new revision (git mv is fine for that).

Reply:
Done, thanks!